eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.js

```diff
@@ -203,29 +292,35 @@ export const UnwrappedNavigationSidebar = (props) => {
             canAddQuestionPage={canAddQuestionPage}
             onAddQuestionConfirmation={onAddQuestionConfirmation}
             canAddQuestionConfirmation={canAddQuestionConfirmation}
+            canAddFolder={canAddFolder}
+            onAddFolder={onAddFolder}
             data-test="nav-section-header"
           />
-          <AccordionGroupToggle
-            onClick={() => handleClick()}
-            data-test="toggle-all-accordions"
-          >
-            {label ? "Close all" : "Open all"}
-          </AccordionGroupToggle>
+          <OpenAllSectionsBtn onClick={() => toggleSections(!openSections)}>
+            {`${openSections ? "Open" : "Close"} all sections`}
```

Comment:
These should be the other way around
```suggestion
            {`${openSections ? "Close" : "Open"} all sections`}
```